addons/binding/org.openhab.binding.opendaikin/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="opendaikin" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ac_unit">
+		<label>Daikin AC Unit</label>
+		<description>Daikin AC Unit</description>
+
+		<channels>
+			<channel id="power" typeId="acunit-power" />
+			<channel id="settemp" typeId="acunit-settemp" />
+			<channel id="indoortemp" typeId="acunit-indoortemp" />
+			<channel id="outdoortemp" typeId="acunit-outdoortemp" />
+			<channel id="humidity" typeId="acunit-humidity" />
+			<channel id="mode" typeId="acunit-mode" />
+			<channel id="fanspeed" typeId="acunit-fan" />
+			<channel id="fandir" typeId="acunit-fandir" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<description>The Host address of the Daikin AC unit.</description>
+				<context>network_address</context>
+			</parameter>
+			<parameter name="refresh" type="integer" required="false">
+				<label>Refresh Interval</label>
+				<description>The number of milliseconds between fetches of the AC unit state.</description>
+				<default>60000</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="acunit-power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power for the AC unit</description>
+	</channel-type>
+
+	<channel-type id="acunit-settemp">
+		<item-type>Number</item-type>
+		<label>Set Point</label>
+		<description>The set point in degrees Celsius or Fahrenheit</description>
+		<category>Temperature</category>
+		<state pattern="%.1f °" step="0.5" />
+		<config-description>
+			<parameter name="useFahrenheit" type="boolean" required="false">
+				<label>Use Fahrenheit</label>
+				<description>Whether to use Fahrenheit or Celsius</description>
+				<default>false</default>
+			</parameter>
+		</config-description>
+	</channel-type>
+
+	<channel-type id="acunit-indoortemp">
+		<item-type>Number</item-type>
```

Comment:
You can use the new QuantityType here: `Number:Temperature` That way you don't have to handle the conversion between Celcius and Fahrenheit. It will be done automatically by openHAB based on the unit the user configures the label with. You can remove the config here (also for the other channels) Remove the conversion code and update the readme.